modules/deployment-model-impl/src/main/java/org/mule/runtime/module/deployment/impl/internal/plugin/PluginExtendedClassLoaderModelAttributes.java

```diff
@@ -9,18 +9,19 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 import org.mule.runtime.module.artifact.api.descriptor.ArtifactDescriptor;
-import org.mule.runtime.module.artifact.internal.classloader.ExtendedClassLoaderModelAttributes;
+import org.mule.runtime.module.artifact.internal.classloader.ExtendedClassLoaderConfigurationAttributes;
 
 import java.util.Map;
 
 /**
- * Allows to extends the attributes defined for a {@link org.mule.runtime.module.artifact.api.descriptor.ClassLoaderModel} when it
- * is being loaded by {@link org.mule.runtime.module.artifact.api.descriptor.ClassLoaderModelLoader} for plugins in order to
- * define in which deployable artifact the plugin is declared.
+ * Allows to extends the attributes defined for a {@link org.mule.runtime.module.artifact.api.descriptor.ClassLoaderConfiguration}
+ * when it is being loaded by {@link org.mule.runtime.module.artifact.api.descriptor.ClassLoaderConfigurationLoader} for plugins
+ * in order to define in which deployable artifact the plugin is declared.
  *
  * @since 4.2.0
  */
-public class PluginExtendedClassLoaderModelAttributes extends ExtendedClassLoaderModelAttributes {
+// TODO - W-11098291: rename accordingly
+public class PluginExtendedClassLoaderModelAttributes extends ExtendedClassLoaderConfigurationAttributes {
```

Comment:
why not this one?

Reply:
Because it's being used in the [`mule-extensions-maven-plugin`](https://github.com/mulesoft/mule-extensions-maven-plugin/blob/d0b7c0d887df53ec5f2b166d0964de3abb2365fd/mule-extensions-maven-plugin/src/main/java/org/mule/extension/maven/loader/MuleArtifactResourcesRegistry.java#L212).

Reply:
should we create a new one and deprecate this then?